torch/__init__.py

```diff
@@ -165,11 +167,57 @@ def _preload_cuda_deps(lib_folder, lib_name):
         raise ValueError(f"{lib_name} not found in the system path {sys.path}")
     ctypes.CDLL(lib_path)
 
-
 # See Note [Global dependencies]
 def _load_global_deps() -> None:
+
+    LIBTORCH_PKG_NAME = "libtorchsplit"
+
+    def find_package_path(package_name):
+        loader = pkgutil.find_loader(package_name)
+        if loader:
+            # The package might be a namespace package, so get_data may fail
+            try:
+                file_path = loader.get_filename()  # type: ignore[attr-defined]
+                return os.path.dirname(file_path)
+            except AttributeError:
+                pass
+        return None
+
+    def load_shared_libraries(library_path):
```

Comment:
A curious q:  The `ctypes.CDLL` call below online 179 doesn't support Windows, but this function does.  What would be the reason for this discrepancy?